apps/pie-storybook/stories/pie-toast.stories.ts

```diff
@@ -1,36 +1,90 @@
 import { html } from 'lit';
-
-/* eslint-disable import/no-duplicates */
-import '@justeattakeaway/pie-toast';
-import { ToastProps } from '@justeattakeaway/pie-toast';
-/* eslint-enable import/no-duplicates */
-
+import { ToastProps, defaultProps } from '@justeattakeaway/pie-toast';
+import { action } from '@storybook/addon-actions';
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
 type ToastStoryMeta = StoryMeta<ToastProps>;
 
-const defaultArgs: ToastProps = {};
+const defaultArgs: ToastProps = {
+    ...defaultProps,
+    message: 'Menu Published',
+    leadingAction: {
+        text: 'Confirm',
+        ariaLabel: 'Descriptive confirmation text',
+    },
+};
 
 const toastStoryMeta: ToastStoryMeta = {
     title: 'Toast',
     component: 'pie-toast',
-    argTypes: {},
+    argTypes: {
+        isOpen: {
+            description: 'When true, the toast is set to be open and visible.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultProps.isOpen,
+            },
+        },
+        isDismissible: {
+            description: 'Allows dismissing the toast by clicking on the close button.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultProps.isDismissible,
+            },
+        },
+        message: {
+            description: 'The message content of the toast.',
+            control: 'text',
+        },
+        isMultiline: {
+            description: 'It allows the message content being displayed as multiline limited to three rows.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultProps.isOpen,
+            },
+        },
+        leadingAction: {
+            description: 'The leading action configuration for the toast.',
+            control: 'object',
+        },
+    },
     args: defaultArgs,
     parameters: {
         design: {
             type: 'figma',
-            url: '',
+            url: 'https://www.figma.com/design/pPSC73rPin4csb8DiK1CRr/branch/sjkhCbCcv7t4QkDrz7g0Xx/%E2%9C%A8-[Core]-Web-Components-[PIE-3]?node-id=7934-12060',
```

Comment:
this design link doesn't work
<img width="1470" alt="Screenshot 2024-07-22 at 12 21 14" src="https://github.com/user-attachments/assets/46a4456c-27fd-4481-bae1-c6b4c5d33e33">